src/visualization/Turi Create Visualization/src/user_interface/src/elements/Annotate/InfiniteScroll/ImageContainer/index.js

```diff
@@ -4,13 +4,18 @@ import { isRegExp } from 'util';
 import { LabelType } from '../../utils';
 
 class ImageContainer extends Component {
-  constructor(props) {
-    super(props);
+  	constructor(props) {
+		super(props);
 	}
 	
-	image_click = () =>{
-		this.props.toggleInfiniteScroll();
-		this.props.updateIncrementalCurrentIndex(this.props.value);
+	image_click = (e) =>{
+		if (e.shiftKey) {
```

Comment:
Which click is this? Is this on the icon above to toggle infinite scroll?

Reply:
This click is the one to shift select multiple items. I'll refactor to follow the specifications that finder uses